src/index.js

```diff
@@ -11,7 +11,7 @@ const {
 const defaultTeam = Team();
 const teamList = teams.length ? teams.map(team => Team(team)) : [];
 
-const createAttachment = (hasMessages, { owner, repo }) => {
+const createAttachment = (hasMessages, { owner, repo, config }) => {
```

Comment:
Who is calling `createAttachment` and are they passing in `config`? I see it's not being passed here:

https://github.com/target/captains-log/blob/4dbc061a109d21e7992d0095ed40e7ab5bc06906/src/index.js#L84

Reply:
Huh, I swore that I passed that change. Thanks for catching that 👍